src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -5,16 +5,69 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import codeu.model.data.User;
+import codeu.model.store.basic.UserStore;
+import java.util.UUID;
+import java.time.Instant;
 
 /**
  * Servlet class responsible for user registration.
  */
 public class RegisterServlet extends HttpServlet {
 
- @Override
- public void doGet(HttpServletRequest request, HttpServletResponse response)
-     throws IOException, ServletException {
+  /**
+  * Store class that gives access to Users.
+  */
+  private UserStore userStore;
 
-   request.getRequestDispatcher("/WEB-INF/view/register.jsp").forward(request, response);
- }
+  /**
+  * Set up state for handling registration-related requests. This method is only called when
+  * running in a server, not when running in a test.
+  */
+  @Override
+  public void init() throws ServletException {
+    super.init();
+    setUserStore(UserStore.getInstance());
+  }
+
+  /**
+  * Sets the UserStore used by this servlet. This function provides a common setup method
+  * for use by the test framework or the servlet's init() function.
+  */
+  void setUserStore(UserStore userStore) {
+    this.userStore = userStore;
+  }
+
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException { 
```

Comment:
Nit/Question for the team: The Google style guide says to wrap lines that are longer than 100 characters: https://google.github.io/styleguide/javaguide.html#s4.4-column-limit
The reason for this is that it keeps things neat and standardized over a large codebase and minimizes how often lines overflow the width of your editor (since scrolling or automatic editor line wrapping can both be kind of weird). Having one standard can also help minimize lines changing and then changing back as people with different settings edit the file, which keeps the history and git blame clear.

Having said that, this is obviously a minor issue, especially for a relatively small project (small compared to all of Google's open source Java, for example).

So, I leave it up to the team--do you think this (or any other style guide points) are worth requiring?

Reply:
Oh this explains what was going on. i think thats a smart idea. I was kinda confused when i was working on it and thought it was some kind of glitch

Reply:
Glad I could explain--thanks for making the change! :smile: One further nit is that the line continuation should be indented *four* spaces rather than two, so it starts further in than the actual method body.